calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -142,6 +142,8 @@
 	
 	private Cache cache = null;
 	
+	private String m_calRefList = "";
```

Comment:
I don't understand that, @ern . Please, could you explain it a bit more, please? Or point me to some related documentation? Or you can also just tell me where should the field be placed and I will change it ASAP. Thanks.